integration_test/logged_test.dart

```diff
@@ -0,0 +1,77 @@
+import 'package:integration_test/integration_test.dart';
+import 'package:wutsi_wallet/main.dart' as app;
+
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+
+Future<void> restoreFlutterError(Future<void> Function() call) async {
+  final originalOnError = FlutterError.onError!;
+  await call();
+  final overriddenOnError = FlutterError.onError!;
+
+  // restore FlutterError.onError
+  FlutterError.onError = (FlutterErrorDetails details) {
+    if (overriddenOnError != originalOnError) overriddenOnError(details);
+    originalOnError(details);
+  };
+}
+
+Future<void> main() async {
+  final binding = IntegrationTestWidgetsFlutterBinding.ensureInitialized();
+  if (binding is LiveTestWidgetsFlutterBinding) {
+    binding.framePolicy = LiveTestWidgetsFlutterBindingFramePolicy.fullyLive;
+  }
+
+  group(('WU-105'), () {
+
+    testWidgets('normal user account update WU-105',
```

Comment:
Make `WU-105` the prefix.
Something like...
```
testWidgets('WU-105: normal user account update'
```

This will be better then reading the report, all `WU-105` test will be grouped together
Please apply this to all your tests